samples/batch/js/node/main.js

```diff
@@ -0,0 +1,333 @@
+    var apiKeyHeader = defaultClient.authentications['apiKeyHeader'];
+    apiKeyHeader.apiKey = '';
+    // Configure API key authorization: apiKeyQuery
+    var apiKeyQuery = defaultClient.authentications['apiKeyQuery'];
+    apiKeyQuery.apiKey = '';
+
+    //Create API Instance
+    //var apiInstance = new SpeechToTextApiV30.DefaultApi();
+
+    const subscriptionKey = "{your-subscription-key-here}";
+    const serviceRegion = "centralindia"; // e.g., "westus"
+    defaultClient.basePath = 'https://centralindia.api.cognitive.microsoft.com/speechtotext/v3.0'.replace(/\/+$/, '');
+
+    // get all transcriptions for the subscription
+     var opts = { 
+        'skip': 56, // Number | Format - int32. Number of transcriptions that will be skipped.
+        'top': 56 // Number | Format - int32. Number of transcriptions that will be included after skipping.
+      };
+      
+      
+      var callback = function(error, data, response) {
+        if (error) {
+          console.error(error);
+        } else {
+          //console.log('API called successfully. Returned data: ' + JSON.stringify(data));
+          pag_files = data.values;
+          console.log(pag_files.length);
+          for (var i = 0; i < pag_files.length; i++){
+            var file_data = pag_files[i];
+            var tid = file_data.self.substring(file_data.self.lastIndexOf("/")+1);
+           console.log (" tid " + tid);
+            if(file_data.status=='Succeeded' || file_data.status == 'Failed'){
+               console.log("status failed or success");
+               var callback = function(error, data, response) {
+                if (error) {
+                  console.error(error);
+                } else {
+                  console.log('API called successfully.');
+                }
+              };
+              apiInstance.deleteTranscription(tid, callback);
+                
+
+            }
+            else{
+                console.log("status running" + file_data.status);
+            }
+        }
+      };
+    }//end else
+      apiInstance.getTranscriptions(opts, callback);
+}//end deleteAllTranscriptions
+
+
+
+function _getTranscriptionFiles()
+{
+    var opts = { 
+        // Transcription | The details of the new transcription.
+            'transcription' : new SpeechToTextApiV30.Transcription()
+        }
+        return opts;
+}
+
+function _transcribeFromSingleBlob(uri){
+    var opts = { 
+        // Transcription | The details of the new transcription.
+            'transcription' : new SpeechToTextApiV30.Transcription()
+        }
+    
+        opts.transcription = 
+        {
+          "contentUrls": [
+          uri
+          ],
+          "properties": {
+          "wordLevelTimestampsEnabled": true
+          },
+          "locale": LOCALE,
+          "displayName": NAME + new Date(),
+          "description" : DESCRIPTION
+        };
+        return opts;
+}//end of _transcribeFromSingleBlob
+
+
+//To be worked on 
```

Comment:
remove?

Reply:
@chlandsi - Have incorporated the changes in the latest code for main.js.